### ISSUE TITLE
default value not start with correct color

### ISSUE THREAD

**Issue by JayricMok** [prscX/react-native-shine-button#6]
Thank again for making this awesome library.  Bellow is the shine button I made:

```
<RNShineButton shape={"heart"} color={"#ffffff"} fillColor={"#FF6C6C" } size={70} value={true}
  onChange={(isLike)=> this._Vibration(isLike)}/>
```
So, my default value of shine button are `true` , but the button are start with color of `#ffffff`, if the default value is `true` color should start with the fillColor `#FF6C6C` , hope can make this changes, thanks a lot

**Comment by prscX** [comment]
Thanks @JayricMok for raising the issue.

I have found the root cause and fixed the same. Can you please install latest v0.0.5 and verify the same.

Please let me know in case you still face the same issue.

Thanks
</ Pranav >

**Comment by ragone** [comment]
I am still seeing this issue - could we reopen this please @prscX ?

**Comment by prscX** [comment]
@ragone: That's strange. I will validate the same.

Thanks for the update

Thanks
</ Pranav >

**Comment by JayricMok** [comment]
I update to v0.0.5 last week, it work

**Comment by prscX** [comment]
Thanks @JayricMok for verification.

Hi @ragone,

Just now I have released `V0.0.8` and verified it is working as expected. Please find below screenshot of the same:

<img width="562" alt="screen shot 2018-04-22 at 1 18 45 pm" src="https://user-images.githubusercontent.com/28862892/39092624-ea6b8e98-462f-11e8-810d-c9faf96c3bea.png">

Can you please verify the installed version of `react-native-shine-button`

Thanks
</ Pranav >

**Comment by prscX** [comment]
Hi @ragone + @JayricMok, 

Hope it is working for you post upgrading the module. Can I please close the issue?

Thanks
</ Pranav >
